packages/sdk/src/evm/common/any-evm-utils.ts

```diff
@@ -944,10 +995,33 @@ export async function encodeConstructorParamsForImplementation(
         }
 
         return nativeTokenWrapperAddress;
+      } else if (p.name && p.name.includes("linkTokenAddress")) {
+        const chainId = (await provider.getNetwork()).chainId;
+        let linkTokenAddress = getLinkTokenByChainId(chainId);
+
+        if (linkTokenAddress === ethers.constants.AddressZero) {
+          throw new Error(
+            "Chainlink LINK token not found on this network. Please provide as a custom param if available.",
```

Comment:
oof another tricky one for any EVM support. gonna need to restrict the chains at the publish level for packVRF